src/objects/token/admin.py

```diff
@@ -104,7 +127,27 @@ def add_view(self, request, form_url="", extra_context=None):
 class PermissionInline(EditInlineAdminMixin, admin.TabularInline):
     model = Permission
     fk_name = "token_auth"
-    fields = ("object_type", "mode", "use_fields", "fields")
+    fields = (
+        "object_type",
+        "mode",
+        "use_fields",
+        "fields",
+        "get_uuid",
+        "get_version",
+    )
+
+    def get_uuid(self, obj):
+        if obj:
+            if object_type := obj.object_type:
+                return object_type.uuid
+
+    def get_version(self, obj):
+        if obj:
+            if object_type := obj.object_type:
+                return object_type.version
+
+    get_version.short_description = "Version"
+    get_uuid.short_description = "UUID"
```

Comment:
please use `admin.display` decorator for it. And it's not just uuid, it's "Object type UUID"
Same is applied for versions.